scintilla/win32/PlatWin.cxx

```diff
@@ -342,7 +342,7 @@ void Window::SetPositionRelative(PRectangle rc, const Window *relativeTo) {
 		const RECT rcMonitor = RectFromPRectangle(rc);
 
 		HMONITOR hMonitor = MonitorFromRect(&rcMonitor, MONITOR_DEFAULTTONEAREST);
-		// If hMonitor is NULL, that's just the main screen anyway.
+		// If hMonitor is NULL, that's just the main screen anyways.

```

Comment:
Just a remark:
Comparing the difference in v5.5.6, it seems we have "fixed" the typo in the comment (though I didn't see it).
It's good to keep the typo as it is, so next update, we don't check this file.

Reply:
Typo change came from https://github.com/notepad-plus-plus/notepad-plus-plus/commit/b742c540bbd68412854783431a8465a63eee9300#diff-90a720dd83a501deb7a8329865ebdefae5156329cc211762ffe9f825dcfa5579 .